app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,11 @@
   <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Indie+Flower" />
 </head>
 <body>
+  <header>
```

Comment:
I really like the way that you decided to include the header in the application view